pkg/kubectl/cmd/set/set_resources.go

```diff
@@ -87,17 +85,11 @@ func NewCmdResources(f cmdutil.Factory, out io.Writer, errOut io.Writer) *cobra.
 		Out: out,
 		Err: errOut,
 	}
-	var pod_specs string
-	RESTMappings := cmdutil.ResourcesWithPodSpecs()
-	for _, Map := range RESTMappings {
-		pod_specs = pod_specs + ", " + Map.Resource
-
-	}
 
 	cmd := &cobra.Command{
 		Use:     "resources (-f FILENAME | TYPE NAME)  ([--limits=LIMITS & --requests=REQUESTS]",
 		Short:   "update resource requests/limits on objects with pod templates",
-		Long:    resources_long + " " + pod_specs[2:],
+		Long:    resources_long,
```

Comment:
Sounds good, but I'd like to make it in a separate one, and get this in quickly.